modules/testing/pages/integration/standardized-outputs.adoc

```diff
@@ -0,0 +1,19 @@
+= Standardized outputs
```

Comment:
Do you want to contain this information here or should we only link to the ADR? 

We can also include a statement about how the results are rendered in the Konflux UI.

Reply:
My plan was to have brief introduction to the outputs so user get the idea of what each one represents, and if they are interested in more detail about it, they can look into the ADR.
Adding the infor about how results are rendered in the Konflux UI is good point, let me add it.